Bottom: 13014b1e4d2262487b0a09c993a6551bee9b30d2
Top:    6f155a817b55a1d4380c1dc69dd45314d1499f68
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-12-10 02:46:40 +0000

Refresh of remove-my-memory-allocation-ro

---

diff --git a/src/rats.c b/src/rats.c
index 2ada575..3c5ede1 100644
--- a/src/rats.c
+++ b/src/rats.c
@@ -727,7 +727,6 @@ AddAllRats (bool SelectedOnly, void (*funcp) ())
   END_LOOP;
   FreeNetListMemory (Nets);
   free (Nets);
-  Nets = NULL;
   FreeConnectionLookupMemory ();
   RestoreFindFlag ();
   if (funcp)
